doc/Language/exceptions.pod6

```diff
@@ -57,18 +58,22 @@ It's possible to handle exceptional circumstances by supplying a C<CATCH> block:
 
     # OUTPUT: «some kind of IO exception was caught!»
 
-Here, we are saying that if any exception of type C<X::IO> occurs, then the
-message C<some kind of IO exception was caught!> will be sent to I<stderr>,
-which is what C<$*ERR.say> does, getting displayed on whatever constitutes the
-standard error device in that moment, which will probably be the console by
-default.
+Here, we are saying that if any exception of type C<X::IO> occurs, then
+the message C<some kind of IO exception was caught!> will be sent to
+I<stderr>, which is what C<$*ERR.say> does, getting displayed on
+whatever constitutes the standard error device in that moment, which
+will probably be the console by default.
 
-A X<C<CATCH>|CATCH> block uses smartmatching similar to how C<given/when>
-smartmatches on options, thus it's possible to catch and handle various
-categories of exceptions inside a C<when> block.
+Note that the match target is a role. To allow user defined exceptions
+to match in the same manner, they must implement the given role. Just
+existing in the same namespace will look alike but won't match in a
```

Comment:
... "make them look alike"?

Reply:
This is a strange paragraph... I think what it's trying to say that other exception types won't match simply because they're in the `X` namespace. I don't think explaining how roles can be used with exceptions really fits on this part of the page.

Reply:
I was saying "will look alike" would be better as "will make them look alike". Not getting into anything else.